OidcDebugger/Views/Debug/Index.cshtml

```diff
@@ -1,150 +1,178 @@
+                        Matches the original state
+                    </span>
+                    <span class="orange-text" v-show="state.value !== expectedState">
+                        <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+                        Does not match the original state: <code>{{expectedState}}</code>
+                    </span>
+                </template>
+            </h2>
+
+            <code-card title="Authorization code" class="success"
+                       v-show="flow === 'code'"
+                       v-bind:value="code.value">
+                {{code.value}}
+            </code-card>
+
+            <code-card title="Access token" class="success"
+                       v-show="accessToken.exists"
+                       v-bind:value="accessToken.value">
+                {{accessToken.value}}
+            </code-card>
+
+            <decoded-token-card title="Access token (decoded)"
+                                class="success"
+                                v-show="accessToken.exists"
+                                v-bind:value="accessToken.value">
+                {{accessToken.value}}
+            </decoded-token-card>
+
+            <code-card title="ID token" class="success"
+                       v-show="idToken.exists"
+                       v-bind:value="idToken.value">
+                {{idToken.value}}
+            </code-card>
+
+            <decoded-token-card title="ID token (decoded)"
+                                class="success"
+                                v-show="idToken.exists"
+                                v-bind:value="idToken.value">
+                {{idToken.value}}
+            </decoded-token-card>
+        </div>
+
+        <div class="debug__callback-header" v-if="!success">
+            <h1 class="red-text">
+                <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+                <b>Error</b>
+            </h1>
+            <h2 class="h4-responsive">
+                The authorization server responded with an error. This could mean that the request parameters were incorrect, or that the user rejected the authorization.
+            </h2>
+
+            <code-card class="error"
+                       v-bind:title="error.value ? 'Error: ' + error.value : 'Error'"
+                       v-bind:value="errorDescription.value">
+                {{safeUnescape(errorDescription.value) || '(no error description)'}}
+            </code-card>
+        </div>
+
+        <div class="debug__callback-info" v-if="success && flow === 'code'">
+            <h3 class="h5-responsive orange-text">Step 2: Exchange code for tokens</h3>
+
+            <div v-if="usePkce">
+                <p>
+                    Now you need to exchange the authorization code for tokens using the token endpoint.
+                    Since you selected to use the PKCE flow we can use the code challenge you created
+                    on the previous page.
```

Comment:
Suggestion:
> Now you need to exchange the authorization code for tokens using the token endpoint. Since this flow was started with a PKCE Code Challenge, we can perform the token exchange directly in the browser for you.

Reply:
@nbarbettini [changed](https://github.com/nbarbettini/oidc-debugger/pull/72/commits/81c05fce8c35e39dd465ac4d5753e47cf3575b85)